### PR TITLE
travis-ci: upgraded to openssl-1.0.2i

### PR DESCRIPTION
**Pull request by chipitsine**

### CODE REVIEW

**Comment by selvanair** [comment]
@mattock: Is this good to merge?

File: .travis.yml [hunk 1]
```diff
@@ -39,7 +39,6 @@ script:
     - ./configure --prefix=/ --libdir=/lib --host=$CHOST --build=x86_64-pc-linux-gnu --program-prefix='' $CONFIG && make
     
 cache:
-  apt: true
```
Comment (mattock) [inline .travis.yml]: Why is this line removed? Is it redundant?

**Comment by mattock** [comment]
Besides the one line comment this looks good.

File: .travis.yml (hunk 1 above)
Comment (chipitsine) [inline .travis.yml]: actually, it does not work (and never supposed to work).
so, I removed it.

Comment (mattock) [inline .travis.yml]: Ok, then I have no further concerns. Merging.
